src/payload-types.ts

```diff
@@ -2535,17 +2571,22 @@ export interface TopBar {
   id: string;
   starText?: {
     desktop?:
-    | {
-      [k: string]: unknown;
-    }[]
-    | null;
+      | {
+          [k: string]: unknown;
+        }[]
+      | null;
     mobile?:
-    | {
-      [k: string]: unknown;
-    }[]
-    | null;
+      | {
+          [k: string]: unknown;
+        }[]
+      | null;
   };
   announcement?: (string | null) | Announcement;
   updatedAt?: string | null;
   createdAt?: string | null;
 }
+
+
+declare module 'payload' {
```

Comment:
@paulpopus Need to remove this section from `payload-types` file - will cause `build` to fail